source/dshow-settings.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "../dshowcapture.hpp"
+#include "dshow-base.hpp"
+#include "json.hpp"
+#include <vector>
```

Comment:
```suggestion
```
Also looks like `vector `is not needed for this include

Reply:
Removed this one too